### PR TITLE
Bug 1558310 Added variables to install docs

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1558310

@openshift/team-documentation PTAL

### CODE REVIEW

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -235,7 +235,7 @@ to `node-role.kubernetes.io/compute=true` if undefined.
 |`openshift_docker_additional_registries`
 a|{product-title} adds the specified additional registry or registries to the
 *docker* configuration. These are the registries to search.
-If the registry required to access the registry is other than `80` include the port number required.
+If the registry required to access the registry is other than `80` include the port number required in the form of `<address>:<port>`.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: s/include/include,
I'd s/port number required/that port number (I think it's a little clearer this way. I think that you can tell from the context that it's required.)

File: install/configuring_inventory_file.adoc [hunk 2]
```diff
@@ -746,6 +746,12 @@ openshift_cockpit_deployer_version=<cockpit_image_tag>
 |Variable |Purpose
 |`openshift_cockpit_deployer_prefix`
 |Specify the URL and path to the directory where the image is located.
+ifdef::openshift-origin[]
+The default value is `kubernetes` so the value is `cockpit/kubernetes:latest`. This differs from several other images that have the `origin-` prefix.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: s/is \`kubernetes\` /is \`kubernetes\`, 
s/differs/format differs
I might s/images/image paths

File: install/configuring_inventory_file.adoc [hunk 3]
```diff
@@ -746,6 +746,12 @@ openshift_cockpit_deployer_version=<cockpit_image_tag>
...
+endif::[]
+ifdef::openshift-enterprise[]
+The value for this must end in `/openshift3` rather than `ose-`, which is the standard for other images.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: s/this/the path

**Review by kalexand-rh — COMMENTED** [review]
Picks and clarifying pronouns. It's sounding good.

**Comment by bfallonf** [comment]
Thanks @kalexand-rh . Merging.

**Pull request merged**